### ISSUE TITLE
Explicate packages needed to use this package

### ISSUE THREAD

**Issue by PhilippKaniuth** [ColinConwell/DeepNSD#1]
In the Google Colab tutorial you link in the readme, `from neural_data import *` cannot work in a conda environment that doesn't have e.g. _gdown_ installed. This is unknown to a user until run. It might help to explicate which packages/modules the user should install before running commands from this package e.g. by uploading a [conda environment file](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually) or a simple requirements.txt.

**Comment by ColinConwell** [comment]
@PhilippKaniuth definitely. this on the to-do list, especially given Colab's tendency to regularly update its packages.
